install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/tasks/build_securedrop_app_code_deb.yml

```diff
@@ -16,9 +16,21 @@
 
 - include: sass.yml
 
+- name: Install Debian packages required to build wheels
+  apt:
+    name:
+      - apache2-dev
+      - python3-dev
+      - python3-pip
+      - python3-wheel
+    state: present
```

Comment:
These py3 dependencies are _in addition_ to the py2 dependencies declared in `molecule/builder-xenial/Dockerfile`. We should instead modify the dependencies in the Dockerfile for the build, push a new image, and store the hash as part of this PR. Happy to append to this PR if you agree. Storing the dependencies in the container image definition will help keep local builds snappy, leveraging the cached layers.

Reply:
I've left this in for now, but as soon as we can get the image updated I'll remove this step.